### ISSUE TITLE
policy cannot be applie to default zone without interface

### ISSUE THREAD

**Issue by sergey-safarov** [firewalld/firewalld#1205]
### What happened

I have firewalld box with one network interface and `dmz` defined by the added network

**/etc/firewalld/zones/dmz.xml**
```xml
<?xml version="1.0" encoding="utf-8"?>
<zone>
  <short>DMZ</short>
  <description>For computers in your demilitarized zone that are publicly-accessible with limited access to your internal network. Only selected incoming connections are accepted.</description>
  <service name="ssh"/>
  <source address="2600:1f1c:910:9106::/64"/>
  <forward/>
</zone>
```
And then configured policy to reach the internet resources from the `dmz` zone
```
firewall-cmd --new-policy dmz-public --permanent
firewall-cmd --reload
firewall-cmd --policy dmz-public --add-ingress-zone=dmz --permanent
firewall-cmd --policy dmz-public --add-egress-zone=public --permanent
firewall-cmd --policy dmz-public --set-target=ACCEPT --permanent
firewall-cmd --reload
firewall-cmd --info-policy dmz-public
```
Then I try ping IP in the Intenet (`public` zone) from server in the `dmz` zone and I got response "Destination unreachable: Administatively prohibited".

### What you expected to happen

I expect that when configured policy like the above and no interface assigned to the zone it should allow traffic to the `default` (public) zone.

### Firewalld Version

1.3.1

### Firewalld Backend

nftables

### Linux distribution

Fedora Linux 38 (Cloud Edition)

### Linux kernel version

6.4.12-200.fc38.aarch64

**Comment by sergey-safarov** [comment]
From my perspective policy rules should be inserted into the `filter_FORWARD_POLICIES_post` chain.
I have checked this chain and it is empty.
```
	chain filter_FORWARD_POLICIES_post {
	}
```

**Comment by sergey-safarov** [comment]
I also have added
```
firewall-cmd \
  --permanent \
  --policy=dmz-public \
  --add-rich-rule='rule family="ipv6" source address="2600:1f1c:910:920d::/64" accept'
```
And then
```
firewall-cmd --reload
nft list ruleset
```
And do not see rule in the `nft` utility output.

**Comment by erig0** [comment]
Hi, I think this is actually fixed in v2.0.0. But the fix is so intensive that it won't be backported to v1.3.z.

Can you try v2.0.0 or later?

**Comment by sergey-safarov** [comment]
yes, I have installed 2.0.0 and ccanconfirm issue resolved.
Expecte rules preent in `nft list ruleset` output.
